src/Command/MigrateToJsonTypeCommand.php

```diff
@@ -37,14 +52,13 @@ public function execute(InputInterface $input, OutputInterface $output)
         $table = $input->getOption('table');
         $column = $input->getOption('column');
         $columnId = $input->getOption('column_id');
-        $connection = $this->getContainer()->get('doctrine.orm.entity_manager')->getConnection();
-        $medias = $connection->fetchAll("SELECT * FROM $table");
+        $medias = $this->entityManager->getConnection()->fetchAll("SELECT * FROM $table");
```

Comment:
It would be nice to check if the entityManager exists and throw an exception if not at the beginning of the execution.

Reply:
https://github.com/sonata-project/SonataMediaBundle/pull/1765/files#diff-7f593d6eca9d1f30aae074420864659bR51-R53 will be enought?